plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/EcmaScriptTranspiler.java

```diff
@@ -87,7 +92,9 @@
 	@Inject
 	private Provider<BlockTransformation> blockTransformationProvider;
 	@Inject
-	private Provider<FormalParameterTransformation> formalParameterTransformationProvider;
+	private Provider<AsyncAwaitTransformation> asyncAwaitTransformationProvider;
+	@Inject
+	private Provider<RestParameterTransformation> restParameterTransformationProvider;
```

Comment:
Rest parameters are an ES6 language feature:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/rest_parameters

Same as what we call "varargs".